api-reference/_hidden/dxDataGridColumn/groupCellTemplate.md

```diff
@@ -60,6 +60,108 @@ The group cell's value with applied [format](/api-reference/_hidden/dxDataGridCo
 The group cell's value as it is specified in a data source.
```

Comment:
Since we shouldn't end sentences with prepositions, I'd like you to make the following changes. (I can't seem to edit these lines directly above.)

Line 16: The settings of the column the cell belongs to.
"The settings of the column to which the cell belongs."

Line 19: The index of the column the cell belongs to.
"The index of the column to which the cell belongs."

Line 50: The index of the row the cell belongs to.
"The index of the row to which the cell belongs."

Reply:
Thank you.